client/src/pages/UpdateGoal.tsx

```diff
@@ -120,41 +122,74 @@ const NotesWrapper = styled.div`
 
 function UpdateGoal() {
   const navigate = useNavigate();
+  const auth = useAuthUser();
+  const username = auth()?.username;
   const [snailName, setSnailName] = useState('');
   const [snailImage, setSnailImage] = useState('');
   const [snailHealth, setSnailHealth] = useState(3);
+  const [notes, setNotes] = useState(' ');
```

Comment:
can you load the notes from the goal data and set it so they display in the textarea?